packages/cursorless-engine/src/processTargets/modifiers/PreferredScopeStage.ts

```diff
@@ -22,6 +22,18 @@ export class PreferredScopeStage implements ModifierStage {
   run(target: Target): Target[] {
     const { scopeType } = this.modifier;
 
+    const containingScopeStage = new ContainingScopeStage(
+      this.modifierStageFactory,
+      this.scopeHandlerFactory,
+      { type: "containingScope", scopeType },
+    );
+
+    try {
+      return containingScopeStage.run(target);
+    } catch (_ex) {
+      // Do nothing. We'll try the closest scope next.
```

Comment:
```suggestion
      // This is expected if one isn't found. We'll move onto closest scope next.
```

Reply:
If there's a more specific exception class that we can catch, that would be preferable, since it would make the exception more readable / the comment redundant (although we still need it because of the linter :P), but also prevent us from wrongly catching something else